test/tasks/Regression.t.sol

```diff
@@ -103,6 +104,29 @@ contract RegressionTest is Test {
         _assertDataToSignSingleMultisig(multisigTask, actions, expectedDataToSign);
     }
 
+    /// @notice expected call data and data to sign generated by manually running the EnableDeputyPauseModuleTemplate at block 7745524 on sepolia
```

Comment:
Small nit on this comment, needs to be for SetRespectedGameType with the correct block on mainnet.